buildkite/bazelci.py

```diff
@@ -979,7 +980,7 @@ def execute_bazel_build(bazel_binary, platform, flags, targets, bep_file, incomp
             "bazel build failed with exit code {}".format(e.returncode))
 
 
-def execute_bazel_test(bazel_binary, platform, flags, targets, bep_file, monitor_flaky_tests, incompatible_flags):
+def execute_bazel_test(bazel_binary, platform, flags, targets, bep_file, monitor_flaky_tests, incompatible_flags, cwd=None):
```

Comment:
I think we should run every command in the given working directory, not just `bazel test`.

The easiest way to achieve this looks like it'd be to just `os.chdir()` here: https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazelci.py#L480

Reply:
Sorry, I would keep this PR smaller.